src/App.js

```diff
@@ -1,7 +1,12 @@
 import "./App.css";
+import Calculator from "./components/calculator.js";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
 }
```

Comment:
Great job so far :+1: However The `App.js` component should be class-based and not a function component, Kindly refactor it into class component.

```

class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```